docs/helm-deployment-minikube.md

```diff
@@ -0,0 +1,127 @@
+# Alfresco Content Services Community Deployment with Helm using Minikube
+
+## Prerequisites
+
+To run the Alfresco Content Services Community deployment using Minikube requires:
+
+| Component   | Getting Started Guide |
+| ------------| --------------------- |
+| Minikube    | https://github.com/kubernetes/minikube |
+| Kubectl     | https://kubernetes.io/docs/tasks/tools/install-kubectl/ |
+| Helm        | https://docs.helm.sh/using_helm/#quickstart-guide |
+
+### Starting Minikube VM
+
+* Allocate at least 16GB RAM (to distribute among the Alfresco Content Services Community cluster nodes), 2 CPU cores, and 20GB disk space to the Minikube VM:
+```bash
+minikube start --cpus=4 --memory=16000
+```
+* Check that Minikube is running:
+```bash
+minikube version
+minikube ip
+```
+* Start the Dashboard to see a web-based user interface of the Minikube cluster:
+```bash
+minikube dashboard
+```
+
+Here's an example start up command for Windows 10. You'll need to have the "My_Virtual_Switch" set up before this first command - see [blog](https://blogs.msdn.microsoft.com/wasimbloch/2017/01/23/setting-up-kubernetes-on-windows10-laptop-with-minikube/):
+```bash
+minikube start --vm-driver="hyperv" --cpus=4 --memory=12000 --hyperv-virtual-switch="My_Virtual_Switch" --v=7 --alsologtostderr
+```
+This downloads a Linux ISO and installs it in your Hyper V Manager. You should see a _minikube_ VM, after it's installed. It also installs all the required software in that VM, to simulate a Kubernetes cluster.  
+You may need to add the ```--extra-config=kubelet.ImagePullProgressDeadline=30m0s``` parameter to your start command, as the docker images are rather big.
+
+**Note:** Although 16GB is the required minimum memory setting, keep in mind that 12GB is lower than the required minimum, and may need to be adapted for your environment.
```

Comment:
**Note:**  You may need to adapt the required RAM requirement to your environment. 12GB may not be enough for any use case.